### ISSUE TITLE
SVC massively overfits on multiclass data sets?

### ISSUE THREAD

**Issue by rhiever** [rhiever/tpot#82]
In one of my [latest commits](https://github.com/rhiever/tpot/commit/1229659cdf65e68caefcfec6a34d7374752b978e#diff-725ecd9a5a19927c516ff00c762c2294R110), I removed the SVC model operator because I noticed it was massively overfitting on the MNIST data set. For example, it would achieve a 99% accuracy on the internal TPOT validation set, but would drop to ~30% accuracy on the external validation set. I'm not sure if this is a known issue with SVC, but I'm simply removing it in the meantime to prevent this from happening.

Hopefully the [sklearn-benchmarks](https://github.com/rhiever/sklearn-benchmarks/) project that we're working on will shed some light on what's going on.
